creator-node/src/routes/nodeSync.js

```diff
@@ -139,7 +139,7 @@ module.exports = function (app) {
    * Given walletPublicKeys array and target creatorNodeEndpoint, will request export
    * of all user data, update DB state accordingly, fetch all files and make them available.
    */
-  app.post('/sync', handleResponse(async (req, res) => {
+  app.post('/sync', authMiddleware, ensurePrimaryMiddleware, handleResponse(async (req, res) => {
```

Comment:
![image](https://user-images.githubusercontent.com/295938/103144878-e4407880-46fe-11eb-964d-b06964e0d793.png)

We probably do not want this route to have ensurePrimaryMiddleware since it's called exclusively by secondaries @SidSethi

Reply:
as far as authMiddleware goes, i don't think session tokens are generated for secondaries in the replica sets

Reply:
lol, good catch - fixed